### PR TITLE
Plugin to view description of npm packages.

### PR DESCRIPTION
**Pull request by mmarcon**
With this plugin it is possible to look at the description and readme for npm packages.

### CODE REVIEW

File: PluginDirectories/1/npm.bundle/info.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	"name": "npmsearch",
```
Comment (nate-parrott) [inline PluginDirectories/1/npm.bundle/info.json]: for future reference, the `name` field needs to be the same as the name of the folder (e.g. `npm`)
I'll fix it for now :)

**Comment by nate-parrott** [comment]
nice! this will be useful.

File: PluginDirectories/1/npm.bundle/info.json (hunk 1 above)
Comment (mmarcon) [inline PluginDirectories/1/npm.bundle/info.json]: Ah, I missed that, sorry. Thanks for fixing it :)

**Comment by nate-parrott** [comment]
Here it is: http://flashlight.nateparrott.com/plugin/npm

**Comment by cojomojo** [comment]
So from first impression I thought the npm-search plugin would search the NPM listings rather than pull up a specific package's description page. IMO it would be better to rename this npm-search to npm-readme or similar and then create an npm-search plugin that searches the npm listings.

**Pull request merged**